.eslintrc.js

```diff
@@ -80,6 +80,7 @@ module.exports = {
 		],
 		'@typescript-eslint/no-var-requires': 'error',
 		'@typescript-eslint/ban-ts-ignore': 0,
+		'@typescript-eslint/explicit-module-boundary-types': 'off',
```

Comment:
This one is new and weird...it seems to give errors in two cases in our code:

1. Where we are using inline functions where the type can be inferred from context. (We already turned down another rule for this.)
2. When we use `any`. We already have the "no explicit any" rule on so this just forces us to add two ignores to the same line when we do which just seems silly to me.

Reply:
For 2, there may be cases where we can start using [unknown](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type). Depending on the number of failures seen, it may be out of scope to fix for this PR.

Reply:
The `any` duplication only shows up in one place in our code, this line:

```
export function applyMixins(derivedCtor: any, baseCtors: any[], options?: MixinOptions): void {
```

We can deal with that by either updating the ignore on the preceding line:

```
// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
```

or, by ignoring the `any` part of this rule:

```
		'@typescript-eslint/explicit-module-boundary-types': [
			'error',
			{ allowArgumentsExplicitlyTypedAsAny: true },
		],
```

For that I would just add the extra disable. The thing that really bothers me if we enable this rule is that we have to start adding types to all of our in-line functions where the type can be inferred from the context. Here's a snippet of a diff wit this rule turned on and all the errors fixed:

```
-		include: app => !!app.webhookSmartApp?.publicKey,
-		value: app => app.webhookSmartApp?.publicKey?.replace(/\r\n/g, '\n') ?? '',
+		include: (app: App): boolean => !!app.webhookSmartApp?.publicKey,
+		value: (app: App): string => app.webhookSmartApp?.publicKey?.replace(/\r\n/g, '\n') ?? '',
```

I don't see how this extra verbosity helps anything.

Reply:
I also think it's strange how this specific case can't be disabled for this one.